README.md

```diff
@@ -99,7 +122,9 @@ Imagine that you take the code from this question, save it to a file called `sus
 
 What would be the output? Explain your answer.
 
-<!-- Replace this comment with your answer -->
+sushi is tasty
+
+The first if statement evaluates to true, given 2 > 0. It does not do the other two checks because they are "else if" and "else" statements, meaning as soon as one of the conditions is true, the rest do not get checked.
```

Comment:
This is still correct; however, it's all the more significant because of `x`. Even though `x` is undefined, this program doesn't give an error because the line evaluating `x` is never reached.